webapp/channels/src/components/post_view/post_reaction/index.ts

```diff
@@ -17,4 +18,4 @@ function mapDispatchToProps(dispatch: Dispatch) {
     };
 }
 
-export default connect(null, mapDispatchToProps)(PostReaction);
+export default connect(null, mapDispatchToProps)(injectIntl(PostReaction));
```

Comment:
Could you change these to use `injectIntl` in the component file instead of in the index? It's kind of arbitrary now that I think of it, but that's how I think we've called this everywhere else